core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -117,9 +117,7 @@
     private static final boolean HAS_WSLENV = System.getenv("WSLENV") != null;
     private static final boolean USE_EPOLL = getBoolean("useEpoll", isEpollAvailable(),
                                                         value -> isEpollAvailable() || !value);
-
-    private static final boolean USE_OPENSSL = getBoolean("useOpenSsl", OpenSsl.isAvailable(),
-                                                          value -> OpenSsl.isAvailable() || !value);
+    private static Boolean USE_OPENSSL = null;
```

Comment:
Because this is not a static final constant anymore, we should use camel case.

Reply:
Could you leave a comment about why we initialize this field lazily?

Reply:
Changed to Camel. Thank you.

Do not call OpenSsl.isAvailable() when not using SSL.
0.2 to 0.3 seconds shortened.

Reply:
could this change be in a patch release with boot 2.2? If it is that much overhead, this will reduce zipkin-slim startup by 15% (I can verify). We are nearing a redo of our release, and it would make the announcement much better I think.

cc @trustin @anuraaga